portfolio/src/main/webapp/style.css

```diff
@@ -93,5 +93,12 @@
 }
 
 .comment {
+    padding: 20px;
+    background-color: white;
+    border-radius: 5px;
     margin-bottom: 10px;
 }
+
+#comment-section {
+    padding-top: 20px;
+}
```

Comment:
Up to you, but it might be good to add `padding-top` to `#comments-container` so that the buttons have a bit of padding from the comments: https://screenshot.googleplex.com/jkN2DKwvVSG